src/main/scala/io.circe.config/package.scala

```diff
@@ -67,11 +67,11 @@ package object config {
   private[config] def jsonToConfigValue(json: Json): ConfigValue =
     json.fold(
       ConfigValueFactory.fromAnyRef(null),
-      boolean => ConfigValueFactory.fromAnyRef(boolean),
+      boolean => ConfigValueFactory.fromAnyRef(boolean.asInstanceOf[java.lang.Boolean]),
```

Comment:
Would it be possible to use the converters in `Predef` instead of casting here?

Reply:
Done!